site/public/templates/misc/MarkdownArea.twig

```diff
@@ -80,6 +80,9 @@
             rows="10" cols="30"
             {# 524288 characters is the default max length for text input fields #}
             maxlength="{{ (no_maxlength|default(null) ? '524288' : (textarea_maxlength|default(null) ? textarea_maxlength :'5000'))}}"
+            {% if required %}
```

Comment:
Could we have:

```
            {% if required is defined and required %}
```

here as well? I get that twig.js doesn't have the same requirement, but would be good to keep them the same as much as possible.

Reply:
I forgot that the two templates have to be kept in sync. I still don't understand why a symlink couldn't be utilized though.

Reply:
Windows has had issues with symlinks for a long time. I think there's been some progress made in the last few years that have really improved things, but looking it up, it does require activating a developer setting in Windows, as well as that some clients may still not work quite right, though I'm not sure.

Easier to avoid the very few symlinks we'd want than field potential issues on getting new developers setup in my opinion.

Reply:
I think a decent alternative would be a script that automatically copies the newer of the two to the same location, and maybe a GitHub Action that ensures the two items have the same hash.